web-app-router/components/LoggedUserNav/HelpNav/index.tsx

```diff
@@ -37,39 +54,48 @@ export const HelpNav = () => {
         </Typography>
 
         <DropdownItem className="w-full hover:bg-grey-50 px-4">
-          <Button
-            href={WORLDCOIN_PRIVACY_URL}
-            className="grid grid-cols-auto/1fr items-center gap-x-2"
-          >
-            <SecurityIcon />
-            <Typography variant={TYPOGRAPHY.R4} className="text-grey-900">
-              Data Privacy & Security
-            </Typography>
-          </Button>
+          <div>
```

Comment:
had to wrap in div otherwise was throwing a forwardref error

Reply:
`<DropdownItem as="div" ...`

Reply:
this then gives me an error since we are wrapping buttons that buttons cannot be divs